### PR TITLE
[SFTP-FIX-4.3] Fix SFTP on ocp 4-3

### PR DESCRIPTION
**Pull request by Stefan365**
//test: `@integrations-sftp-to-sftp`

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

File: utilities/src/main/java/io/syndesis/qe/bdd/validation/SftpValidationSteps.java [hunk 1]
```diff
@@ -29,6 +29,12 @@ public void validateThatFileHasBeenTransferedFromToSftpDirectory(String filename
         assertThat(sftpUtils.isFileThere(serverFromDirectory, filename)).isFalse();
     }
 
+    @Then("^check \"([^\"]*)\" is in \"([^\"]*)\" directory with content \"([^\"]*)\" using SFTP$")
+    public void getFileContent(String fileName, String directory, String text) {
+        String content = sftpUtils.getFileContent(directory, fileName);
+        assertThat(text).isEqualTo(content);
```
Comment (avano) [inline utilities/src/main/java/io/syndesis/qe/bdd/validation/SftpValidationSteps.java]: `assertThat(text).isEqualTo(sftpUtils.getFileContent(directory, fileName))`

File: utilities/src/main/java/io/syndesis/qe/resource/impl/SFTP.java [hunk 2]
```diff
@@ -51,6 +51,7 @@ public void deploy() {
             .done();
         OpenShiftUtils.getInstance()
             .addRoleToUser("anyuid", "system:serviceaccount:" + TestConfiguration.openShiftNamespace() + ":" + serviceAccountName);
+        OpenShiftUtils.binary().execute("adm", "policy", "add-scc-to-group", "anyuid", "system:authenticated");
```
Comment (avano) [inline utilities/src/main/java/io/syndesis/qe/resource/impl/SFTP.java]: this way you are changing the behavior for all users in the cluster while the sftp is running, so affecting other tests running in different namespaces.

File: utilities/src/main/java/io/syndesis/qe/utils/SftpUtils.java [hunk 3]
```diff
@@ -75,6 +74,7 @@ public void uploadTestFile(String testFileName, String text, String remoteDirect
         try (FileOutputStream out = new FileOutputStream(testFileName);) {
             out.write(text.getBytes("UTF-8"));
             out.close();
+            TestUtils.sleepIgnoreInterrupt(1000);
```
Comment (avano) [inline utilities/src/main/java/io/syndesis/qe/utils/SftpUtils.java]: why?

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Pull request merged**